seleniumByTechnocredits/src/seleniumPractise/pallavi/Alert_Verification.java

```diff
@@ -0,0 +1,35 @@
+package seleniumPractise.pallavi;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import util.Init;
+
+public class Alert_Verification 
+{
+
+	public static void main(String[] args) throws InterruptedException 
+	{
+		WebDriver driver =Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/#");
+		driver.manage().window().maximize();
+		driver.findElement(By.linkText("Basic Elements")).click();
+		Thread.sleep(1000);
+		String expectedResult = "You must be TechnoCredits student!!";
+		driver.findElement(By.xpath(".//button[text()='Alert']")).click();
```

Comment:
Hi Pallavi, Its giving below error when I ran your code:
Exception in thread "main" org.openqa.selenium.WebDriverException: unknown error: Element <button type="submit" class="btn btn-success" id="javascript..." onclick="callJavaScriptAlert()">Alert</button> is not clickable at point (73, 580).

You can write the code to scroll till element before clicking.
Same happened for next two programs.